app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ReviewsController < ApplicationController
+
+  def index
+    restaurant_id = params.require(:restaurant_id)
+    render json: Review.where(:restaurant_id => restaurant_id)
+  end
+
+  def show
+    render json: Review.find(params[:id])
+  rescue ActiveRecord::RecordNotFound => error
+    render json: {status: 'error', code: 404, message: error.message}, :status => :not_found
+  end
+
+  def create
+    render json: Review.create!(review_params), status: :created
+  rescue ActiveRecord::RecordInvalid => error
+    render json: {status: 'error', code: 400, message: error.message}, :status => 400
```

Comment:
please convert all statuses to string symbols
400 --> bad_request

Reply:
better yet, remove all rescues. Rails gives this to you out of the box.